idl/internal/lex.rl

```diff
@@ -283,6 +283,7 @@ func (lex *lexer) Lex(out *yySymType) int {
                 base := 10
                 if len(str) > 2 && str[0:2] == "0x" {
                     // Hex constant
+                    str = str[2:]
```

Comment:
Would you mind adding a comment here indicating why we're doing this? Because this is a bit subtle. Perhaps something like:

```
// Hex constant.
// strconv.ParseInt doesn't want the "0x" prefix if we explicitly set base to 16.
```